src/clang-c-frontend/clang_c_language.cpp

```diff
@@ -262,6 +262,7 @@ std::string clang_c_languaget::internal_additions()
   std::string intrinsics =
     R"(
 # 1 "esbmc_intrinsics.h" 1
+typedef long time_t;
```

Comment:
I don't think this is correct, as it is platform dependent: https://en.cppreference.com/w/c/chrono/time_t

I suggest removing both `nondet_time_t()` and `__VERIFIER_nondet_time_t()` and leave it as it was before: just define it in the time.c model lib.

@lucasccordeiro, do you agree?

Reply:
I agree with you.

I had previously suggested adding the prototype of this function in:

- https://github.com/esbmc/esbmc/blob/master/src/clang-c-frontend/clang_c_language.cpp#L319
- https://github.com/esbmc/esbmc/blob/master/src/ansi-c/ansi_c_language.cpp#L75

Thales told me that he had compilation problems. However, if we look at the spec, we can see that it is platform dependent.

@thales: Could you please remove the declaration of these functions from the files above?

Many thanks,
Lucas

Reply:
Sure!